.travis/check-pr-no-readme.sh

```diff
@@ -4,7 +4,8 @@ set -Eeuo pipefail
 cd "$(dirname "$(readlink -f "$BASH_SOURCE")")/.."
 
 if [ "$TRAVIS_PULL_REQUEST" != 'false' ]; then
-	if [ "$(git diff --numstat "$TRAVIS_COMMIT_RANGE" -- '*/README.md')" ]; then
+	git fetch -q https://github.com/docker-library/docs.git master
+	if [ -n "$(git diff --numstat FETCH_HEAD...HEAD -- '*/README.md')" ]; then
```

Comment:
Woot, this worked, thanks for being a guinea pig for this fix. :innocent: :heart: